lib/ioki/apis/endpoints/delete_singular.rb

```diff
@@ -6,6 +6,27 @@ class DeleteSingular < Delete
       def full_path
         base_path + [resource_path_name]
       end
+
+      def call(client, model, args = [], options = {})
+        outgoing_model_class = @outgoing_model_class || options[:outgoing_model_class] || model_class
+
+        unless model.is_a?(outgoing_model_class)
+          raise(ArgumentError, "#{model} is not an instance of #{outgoing_model_class}")
+        end
+
+        parsed_response, = client.request(
+          url:    client.build_request_url(*Endpoints.url_elements(full_path, *args)),
+          method: :delete,
+          body:   { data: model.serialize(:delete) },
```

Comment:
I think it would be great if we could still use that endpoint class without a body. We'd need to set `body` to `nil`, if the `model` is not set. What do you think?